eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/CBSChannel.java

```diff
@@ -85,4 +50,50 @@ public void close(
 
         this.innerChannel.close(reactorDispatcher, closeCallback);
     }
+
+    private static class RequestResponseChannelOperationResult implements OperationResult<RequestResponseChannel, Exception> {
+        Message requestMessage;
```

Comment:
These member variables can be final because they are set once. Same with the other nested inner class.